packages/runner/cypress/integration/runner.ui.spec.js

```diff
@@ -341,5 +341,25 @@ describe('src/cypress/runner', () => {
         },
       })
     })
+
+    it('supports disabling command log reporter with env var NO_COMMANDLOG', () => {
+      runIsolatedCypress(() => {
+        it('foo', () => {
+          // simulate a page load, ensures reporter state event is properly stubbed
+          cy.then(() => Cypress.action('cy:collect:run:state'))
+          cy.visit('/')
+
+          // ensures runner doesn't wait for nonexist before:screenshot ack
+          cy.screenshot({
+            capture: 'runner',
+          })
+        })
+      },
+      {
+        config: { env: { NO_COMMANDLOG: '1' } },
+      })
+
+      cy.get('.reporter').should('not.exist')
```

Comment:
is there an inverse test case somewhere?

```js
      cy.get('.reporter').should('exist')
```

Reply:
![image](https://user-images.githubusercontent.com/14625260/94744524-1bdd9580-0347-11eb-8f40-97187dd47659.png)

does this count

Reply:
yeeees